config/routes.rb

```diff
@@ -1,7 +1,17 @@
 Myflix::Application.routes.draw do
+
+  root 'static#landing_page'
   get 'ui(/:action)', controller: 'ui'
+  resources :users, only: [:new,:create]
+  get '/sign-in', to: 'sessions#sign_in', as: 'sign_in'
+  post '/checking_credentials', to: 'sessions#checking_credentials'
+  get '/logout', to: "sessions#logout"
 
   get "/home", to: "video#home"
-  get "/video/:id", to: "video#show", as: 'show_video'
   get "/category/:id", to: "category#show", as: 'show_category'
```

Comment:
Any reason why you are not using resources here?

```ruby
resources :category, only: [:show]
```

That would do it for you, and is easier to maintain and build on going forward.